Chargebee/Classes/Network/CBNetworkRequest.swift

```diff
@@ -22,6 +22,10 @@ extension CBNetworkRequest {
                 onError?(CBError.defaultSytemError(statusCode: 400, message: error.localizedDescription))
                 return
             }
+            if let response = response as? HTTPURLResponse, response.statusCode >= 500 {
+                onError?(self.buildCBError(data, statusCode: response.statusCode))
+                return
+            }
```

Comment:
Do we need this for any particular reason? 
The check for `response.statusCode >= 400` should cover this if we do not want to do anything different here

Reply:
For `500` or `serverNoResponding` I added this but the enum case in the error block will be sufficient so this is not required removed it thanks.